x-pack/platform/plugins/shared/stack_connectors/public/common/auth/translations.ts

```diff
@@ -157,3 +157,94 @@ export const EDIT_CA_CALLOUT = i18n.translate(
       'This connector has an existing certificate authority file. Upload a new one to replace it.',
   }
 );
+
+export const ACCESS_TOKEN_URL = i18n.translate(
+  'xpack.stackConnectors.components.auth.accessTokenUrlTextFieldLabel',
+  {
+    defaultMessage: 'Access token URL',
+    description: 'The URL to get the access token from',
+  }
+);
+export const ACCESS_TOKEN_URL_REQUIRED = i18n.translate(
+  'xpack.stackConnectors.components.auth.error.requiredAccessTokenUrlText',
+  {
+    defaultMessage: 'Access token URL is required.',
+  }
+);
+export const CLIENT_ID = i18n.translate(
+  'xpack.stackConnectors.components.auth.clientIdTextFieldLabel',
+  {
+    defaultMessage: 'Client ID',
+    description: 'The client ID to use for AOuth2 authentication',
+  }
+);
+export const CLIENT_ID_REQUIRED = i18n.translate(
+  'xpack.stackConnectors.components.auth.error.requiredClientIdText',
+  {
+    defaultMessage: 'Client ID is required.',
+  }
+);
+export const CLIENT_SECRET = i18n.translate(
+  'xpack.stackConnectors.components.auth.clientSecretTextFieldLabel',
+  {
+    defaultMessage: 'Client secret',
+    description: 'The client secret to use for for AOuth2 authentication',
+  }
+);
+export const CLIENT_SECRET_REQUIRED = i18n.translate(
+  'xpack.stackConnectors.components.auth.error.requiredClientSecretText',
+  {
+    defaultMessage: 'Client secret is required.',
+  }
+);
+export const SCOPE = i18n.translate(
+  'xpack.stackConnectors.components.auth.clientSecretTextFieldLabel',
+  {
+    defaultMessage: 'Scope',
+    description: 'The scope to use for for AOuth2 authentication',
+  }
+);
+
+export const SCOPE_HELP_TEXT = i18n.translate(
+  'xpack.stackConnectors.components.auth.scopeHelpText',
+  {
+    defaultMessage: "Space-separated list of scopes (e.g., 'email profile').",
+  }
+);
+
+export const AUTHENTICATION_OAUTH2 = i18n.translate(
+  'xpack.stackConnectors.components.auth.authenticationMethodOAuth2Label',
+  {
+    defaultMessage: 'OAuth 2.0',
```

Comment:
I'm tempted to update this to `OAuth 2.0 - Client Credentials` to make it clear that it's the only type we allow, wdyt? It's the label of the collapsible inside the create/update connector form

<img width="721" height="291" alt="Screenshot 2025-08-01 at 13 05 28" src="https://github.com/user-attachments/assets/48a88461-ea86-4eb1-8597-4246d6275b31" />